services/notification/notifiers/status/base.py

```diff
@@ -103,11 +106,10 @@ def flag_coverage_was_uploaded(self, comparison) -> bool:
             len(report_uploaded_flags.intersection(flags_included_in_status_check)) > 0
         )
 
-    @property
-    def repository_service(self):
+    def repository_service(self, commit: Commit):
```

Comment:
Let's also add a type annotation for the return value and `self._repository_service`.